src/java.base/share/classes/java/nio/charset/Charset.java

```diff
@@ -628,6 +629,7 @@ public SortedMap<String,Charset> run() {
             });
     }
 
+    @Stable
     private static volatile Charset defaultCharset;
```

Comment:
If the field is stable then does the volatile go away?

Reply:
Right. It should.